ebmdatalab/generate-ebmdatalab-stats.py

```diff
@@ -104,7 +104,7 @@ def assign_ISO(countries):
             else:
                 failed_matches.append(country)
     # Print warning about failures and return successes as dictionary
-    print("Could not assign country codes to: ", ", ".join(failed_matches))
+    print("Could not assign country codes to:", ", ".join(failed_matches))
```

Comment:
```suggestion
    print("Could not assign country codes to: ", ", ".join(failed_matches))
```
This was your fix that seems to have been reverted!

Reply:
I reverted it intentionally because when I ran locally I got two spaces.  It doesn't really matter either way.  It's only a log message in the CI build.

Reply:
Oh no!!! I'm sorry for changing it back. When I push the fix for the ISO codes, I can put this back the way it was!